Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ however, it has to be formatted properly to pass verification tests.
 ### Fixed
 
 - Fixed precompiled layouts such as `FastKeyboard` leading to build time regressions with il2cpp (case 1283676).
+- Fixed update/switch and restart prompt on Unity 2020.2+ ([case 1292513](https://issuetracker.unity3d.com/issues/input-system-after-package-install-the-update-slash-switch-and-restart-prompt-does-not-appear)).
```

Comment:
Suggest: "Fixed restart prompt after package installation not appearing on Unity 2020.2+ (...)."

Reply:
fixed